src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -203,11 +203,7 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
         <ExperimentTopStories topStoriesContent={topStoriesContent} />
       ) : null,
     jumpTo: (props: ComponentToRenderProps & JumpToProps) =>
-      jumpToVariation === 'on_off' ? (
-        <JumpTo {...props} />
-      ) : (
-        <JumpTo {...props} />
-      ),
+      jumpToVariation === 'on' ? <JumpTo {...props} /> : <JumpTo {...props} />,
```

Comment:
Looks good 👍 

Thinking about it, maybe we should just go the whole way now and return `null` if its not `on`.

We can force ourselves into the experiment and toggle the variations to test that it shows/hides correctly. So probably best to do that now too.